src/main/webapp/script.js

```diff
@@ -14,6 +14,21 @@ function registerServiceWorker() {
   }
 }
 
+// Maps the numerical value of the state to data used to create
+// needed to create geoJson map of the state.
+const states = {
```

Comment:
ok for now but we should move this into stateinfo js file

Reply:
Great point! Done.